Makefile

```diff
@@ -78,7 +78,7 @@ clean:
 	@find . -name "*.pyc" -delete
 
 dev-venv: clean ## Create a local venv for your IDE and remote debugging
-	python3.9 -m venv venv
+	python3 -m venv venv
```

Comment:
This is ambiguous and leads to different outcomes on different machines. If we want to upgrade to 3.10, then lets be explicit:

```sh
python3.10 -m venv venv
```

Reply:
isn't it common that python3 is an alias to whatever the user chooses?

Reply:
Absolutely. Which is what makes it ambiguous.

If we say we use python3.10, then we should enforce our dev-env to be 3.10.